ipa-core/src/net/server/handlers/query/metrics.rs

```diff
@@ -0,0 +1,67 @@
+use axum::{routing::get, Router};
+use hyper::StatusCode;
+use opentelemetry::KeyValue;
+
+use crate::net::{
+    http_serde::{self},
+    Error,
+};
+
+use opentelemetry::metrics::MeterProvider;
+use opentelemetry_sdk::metrics::SdkMeterProvider;
+use prometheus::{self, Encoder, TextEncoder};
+
+/// Takes details from the HTTP request and creates a `[TransportCommand]::CreateQuery` that is sent
+/// to the [`HttpTransport`].
+async fn handler(// transport: Extension<MpcHttpTransport>,
+    // QueryConfigQueryParams(query_config): QueryConfigQueryParams,
+) -> Result<Vec<u8>, Error> {
+    // match transport.dispatch(query_config, BodyStream::empty()).await {
+    //     Ok(resp) => Ok(Json(resp.try_into()?)),
+    //     Err(err @ ApiError::NewQuery(NewQueryError::State { .. })) => {
+    //         Err(Error::application(StatusCode::CONFLICT, err))
+    //     }
+    //     Err(err) => Err(Error::application(StatusCode::INTERNAL_SERVER_ERROR, err)),
+    // }
+
+    // TODO: Remove this dummy metrics and get metrics for scraper from ipa-metrics::PrometheusMetricsExporter (see ipa-metrics/exporter.rs)
```

Comment:
@akoshelev I'm still not clear how we can pull the metrics from the logging_handle. IpaHttpClient doesn't seem to have a reference to it?

Reply:
that's right, it is not referenced from anywhere. `helper.rs` does have [logging_handle](https://github.com/private-attribution/ipa/blob/8c201ae1b955da2e38f6a0c634e7a23925e834a3/ipa-core/src/bin/helper.rs#L298) that has a reference to this controller. So we need to plumb it through to deliver `Controller` to the handlers